### PR TITLE
Nanu's Donator Armor

### PR DESCRIPTION
**Pull request by Ashvor**
:cl: Ashvor
rscadd: Nanu's Donator ODST Armor has been added.
tweak: Mann's Donator ODST Armor Sprites updated.
tweak: Sleepy's Magnum given new hitsound.
/:cl:

### CODE REVIEW

File: code/_helpers/unsorted.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@
 	dx=(32*end.x+end.pixel_x)-(32*start.x+start.pixel_x)
 	if(!dy)
 		return (dx>=0)?90:270
-	.=arctan(dx/dy)
+	//.=arctan(dx/dy)
```
Comment (X0-11) [inline code/_helpers/unsorted.dm]: Why is this change here

**Review by X0-11 — DISMISSED** [review]

File: code/_helpers/unsorted.dm (hunk 1 above)
Comment (bloxgate) [reply code/_helpers/unsorted.dm]: I'm going to hazard a guess that they're not using 513 for development. This line when unmodified would cause an error in 512 or below. @Ashvor can you undo this change please?

**Review by bloxgate — DISMISSED** [review]

File: code/_helpers/unsorted.dm (hunk 1 above)
Comment (Ashvor) [reply code/_helpers/unsorted.dm]: Uh sure, but when I updated my fork and tried to compile it gave me an error regarding that line of code. Undefined variable I believe.

Comment (Ashvor) [reply code/_helpers/unsorted.dm]: Fixed your silly little code wonk.

**Review by BDpuffy420 — APPROVED** [review]
